app/models/job_action.rb

```diff
@@ -0,0 +1,25 @@
+class JobAction < ApplicationRecord
+  include Authorizable
+
+  belongs_to :job_template
+  belongs_to :user
+  before_validation :set_user
+
+  validates :name, :job_template, :user, presence: true
```

Comment:
maybe unique name?
If we are keeping user, than unique name per user (in scope of user)